frontend/src/scenes/feature-flags/featureFlagsLogic.ts

```diff
@@ -16,6 +16,7 @@ export enum FeatureFlagsTab {
     Analysis = 'analysis',
     USAGE = 'usage',
     PERMISSIONS = 'permissions',
+    PROJECTS = 'projects',
```

Comment:
We can change this later on but I think 'projects' makes me think of feature flag projects (??) when the concept we want more is like... feature flag environments xD or like... feature flag syncs...

Reply:
idk, non blocking right now, but it'll be confusing later on IMO

Reply:
I agree, on the other hand users already use projects as environments, so a discrepancy in naming might be confusing here. But happy to discuss it, we can always adjust.